spec/factories/result_comment.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
```

Comment:
Factroy should be in plural. So `result_comments.rb` should be better and easier to note the difference to model.

Reply:
:+1: